src/controller/OpenRobertaController.vala

```diff
@@ -75,6 +75,7 @@ namespace BrickManager {
                     });
                 }, () => {
                     status_bar_item.connected = false;
+                    open_roberta_window.connected = false;
```

Comment:
> Not sure what the timelines for releasing all this is.

ASAP, I hope. I've been trying to do an image release for over a month, but there are so many changes, it is taking a long time. I think with the commits I just pushed and these changes, brickman is finally in a releasable state again. But I have to do a kernel release first because brickman/ev3devKit depend on some unreleased kernel changes.